addons/sale_project/models/project.py

```diff
@@ -614,7 +614,7 @@ class ProjectTask(models.Model):
         compute='_compute_sale_line', store=True, readonly=False,
         domain="""[
             ('company_id', '=', company_id),
-            '|', ('order_partner_id', 'child_of', partner_id if partner_id else []),
+            '|', ('order_partner_id', 'child_of', commercial_partner_id if partner_id else []),
```

Comment:
if this is done, shouldn't it be:

```suggestion
            '|', ('order_partner_id', 'child_of', commercial_partner_id if commercial_partner_id else []),
```
(not sure if we can use `or` instead to make it less "mouthy", this might be evaluated in JS so maybe not)

in vanilla case, I guess if `partner_id` is set, `commercial_partner_id` will be set but maybe at one point (or with customization) it could not be the case, so it might make more sense to just test what we are giving